app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id="<%= dom_id group %>" class="groupContainer">
+  <img src=<%= group.Icon %> alt="icon">
+  <div class="linkBox">
+    <%= link_to group.Name, group_expenses_path(group.id) %>
+  </div>
+  <% @total = 0 %>
+  <% group.groupsexpenses.each do |exp| %>
+    <% @total += exp.expenses.Amount %>
+  <% end %>
+  <div class="total"> <strong><%= @total %> $ </strong></div>
+</div>
```

Comment:
- [ ] just as previous reviewer mentioned, kindly remove the text decoration from the group name and make the color look like the one in the design
your design :point_down: 
![Screenshot from 2023-02-16 08-46-12](https://user-images.githubusercontent.com/66097371/219301376-a8a2af29-8931-4d89-96c1-a961491e9838.png)